addons/misc/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -1,6 +1,8 @@
 class ACE_Medical_Treatment_Actions {
+    class FieldDressing;
```

Comment:
Why add FieldDressing here?

Reply:
Left over from testing, totally should be removed good find